tests/scripts/validate_modified_files.sh

```diff
@@ -9,26 +9,31 @@ MOD_ERR="changes found by mod.check. You may need to run make clean"
 CRD_ERR="changes found by 'make crds'. please run 'make crds' locally and update your PR"
 BUILD_ERR="changes found by make build', please commit your go.sum or other changed files"
 HELM_ERR="changes found by 'make gen-rbac'. please run 'make gen-rbac' locally and update your PR"
+DOCS_ERR="changes found by 'make docs'. please run 'make docs' locally and update your PR"
+
 
 #############
 # FUNCTIONS #
 #############
 function validate(){
-  git=$(git status --porcelain)
-  for file in $git; do
-    if [ -n "$file" ]; then
-      echo "$1"
-      echo "$git"
-      git diff
+  ERR=$1
+  git diff --exit-code
```

Comment:
(Nothing to fix but just an observation) Earlier, the function would only run `git status --porcelain` to get the changed files and then print those files. Now it runs `git diff --exit-code` as well as `git status --porcelain`. So we are making an extra git call in the refracted function.

Reply:
closing this.